templates/base.html.twig

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
is this file needed ?

Reply:
in dev mode yes it's needed because the WebProfilerBundle use twig as a template engine

Reply:
WebProfilerBundle does not use this one ? https://github.com/symfony/web-profiler-bundle/blob/master/Resources/views/Profiler/base.html.twig

Reply:
yes you are right, there is no need from that file